src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -64,6 +64,8 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <NoWarn>$(NoWarn);CS8002;NU5131;</NoWarn>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- VS 2019 uses C# 10. Not all C# 10 language features are available. See https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version -->
+    <LangVersion>latestMajor</LangVersion> 
```

Comment:
- Should be `latest`?
- We also specify lang version 7.2 in Directory.Build.props. Should update there?
- But this does mean that not all of the newer features are available for older frameworks? 
- Tidbit from the doc to keep in mind:

> Choosing a language version newer than the default can cause hard to diagnose compile-time and runtime errors.

Reply:
latestMajor is the default I think. I did not realize we have 7.2 in build.props, let me see if I can remove it.